libs/package_manager.py

```
@@ -19,14 +19,20 @@ def _run_command(self, args):
-				# Convert args list to a single command string for shell=True
-				command_string = subprocess.list2cmdline(args)
-				return subprocess.check_call(command_string, shell=True)
+
+				# Use shutil.which to resolve .cmd/.bat extensions without shell=True
+				import shutil
+				cmd = args[0]
+				resolved_cmd = shutil.which(cmd)
+				if resolved_cmd:
+					args[0] = resolved_cmd
+
+				return subprocess.check_call(args, shell=False)
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)